ports/stm32/modnwwiznet5k.c

```diff
@@ -191,11 +191,8 @@ STATIC int wiznet5k_socket_accept(mod_network_socket_obj_t *socket, mod_network_
             socket->u_param.fileno = -1;
             int _errno2;
             if (wiznet5k_socket_socket(socket, &_errno2) != 0) {
-                //printf("(bad resocket %d)\n", _errno2);
             } else if (wiznet5k_socket_bind(socket, NULL, *port, &_errno2) != 0) {
-                //printf("(bad rebind %d)\n", _errno2);
             } else if (wiznet5k_socket_listen(socket, 0, &_errno2) != 0) {
-                //printf("(bad relisten %d)\n", _errno2);
```

Comment:
Are you planning to leave the empty if statements?

Reply:
I guess, what would ba another option?

Reply:
removing them...

Reply:
I'm hesitant to do that since above it states `TODO handle errors` so I assume at one point or another these empty statements are going to get filled with `return -1` or so?